pkg/grpc/labelstore/client/client.go

```diff
@@ -29,11 +29,22 @@ func NewClient(conn *grpc.ClientConn, logger logging.Logger) Client {
 	}
 }
 
+// this interface is just to make the compiler assert that our functions match
+// those in the direct consul applicator
+type client interface {
+	WatchMatches(selector klabels.Selector, labelType labels.Type, _ time.Duration, quitCh <-chan struct{}) (chan []labels.Labeled, error)
```

Comment:
i added a second assertion on line 40 that will make sure that those two stay in sync